packages/nextjs/src/pages/ClerkProvider.tsx

```diff
@@ -9,11 +9,9 @@ import { invalidateNextRouterCache } from './invalidateNextRouterCache';
 // TODO: Import from shared once [JS-118] is done
 const useSafeLayoutEffect = typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
 
-__internal__setErrorThrowerOptions({
-  packageName: '@clerk/nextjs',
-});
+__internal__setErrorThrowerOptions({ packageName: '@clerk/nextjs' });
 
-export * from '@clerk/clerk-react';
+// export * from '@clerk/clerk-react';
```

Comment:
❓ is this intentional? If so, let's remove it instead of comment it out.

Reply:
Its not, will be removed, thanks Dimitri